test/Cle.CodeGeneration.UnitTests/X64CodeGenerator/VariableTests.cs

```diff
@@ -0,0 +1,130 @@
+using NUnit.Framework;
+
+namespace Cle.CodeGeneration.UnitTests.X64CodeGenerator
+{
+    public class VariableTests : X64CodeGeneratorTestBase
+    {
+        [Test]
+        public void Constant_integer_load_and_return()
+        {
+            // return 42;
+            const string source = @"
+; #0   int32
+BB_0:
+    Load 42 -> #0
+    Return #0
+";
+            const string expected = @"
+; Test::Method
+LB_0:
+    mov eax, 2Ah
```

Comment:
Remember to change the hexadecimal syntax to `0x2A` instead of `2Ah`. The latter syntax is used e.g. by Microsoft `dumpbin`, but I guess the former is more familiar.